src/Umbraco.Core/Models/ContentVariation.cs

```diff
@@ -3,34 +3,35 @@
 namespace Umbraco.Core.Models
 {
     /// <summary>
-    /// Indicates the values accepted by a property.
+    /// Indicates how values can vary.
     /// </summary>
+    /// <remarks>
+    /// <para>Values can vary by nothing, or culture, or segment, or both.</para>
+    /// <para>Varying by culture implies that each culture version of a document can
+    /// be available or not, and published or not, individually. Varying by segment
+    /// is a property-level thing.</para>
+    /// </remarks>
     [Flags]
```

Comment:
If a content variation should only be one of these items, shouldn't Flags be removed? A content variation cannot be both Nothing and Something ?

Reply:
should be gone indeed, it's not a true flag enum, will remove